client/src/Booking.jsx

```diff
@@ -47,7 +47,13 @@ class Booking extends React.Component {
   }
 
   init() {
-    axios.get('http://localhost:3002/booking')
+    let url = '';
+    if (process.env.NODE_ENV === 'production') {
+      url += 'http://67.160.218.95/32/booking';
+    } else if (process.env.NODE_ENV === 'development') {
+      url += 'http://localhost:3002/booking'
+    }
+    axios.get(url)
```

Comment:
I did something similar. Ypu may want to consider moving the url's out to a config file in the case that you need to change them in multiple places (like i did)